dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/UploadStep.java

```diff
@@ -87,8 +98,28 @@ public void doPatchProcessing(Context context, Request currentRequest, InProgres
             }
         }
         PatchOperation<?> patchOperation = new PatchOperationFactory().instanceOf(instance, op.getOp());
-        patchOperation.perform(context, currentRequest, source, op);
+        if (instance.equals(AbstractRestProcessingStep.UPLOAD_STEP_METADATA_OPERATION_ENTRY)) {
+            DCInputSet inputConfig = inputReader.getInputsByFormName(UploadStep.UPLOAD_STEP_METADATA_SECTION);
+            String[] split = patchOperation.getAbsolutePath(op.getPath()).split("/");
+            String metadata = findMetadata(split);
+            if (inputConfig.isFieldPresent(metadata)) {
+                patchOperation.perform(context, currentRequest, source, op);
+            } else {
+                throw new UnprocessableEntityException("The attribute " + metadata + " does not present in section "
```

Comment:
Again, likely should say: `"The field " + metadata + " is not present in section "`